README.md

```diff
@@ -0,0 +1,61 @@
+# Google Code-in Leaders
+
+***
+It is a website to show the current leaders from all organizations in GCI.
+***
+
+ * [Installation](https://github.com/learneradarsh/gci-leaders#installation)
```

Comment:
These links point to your repository, which means we cant maintain it.

These links should be relative, pointing to the anchor only, so that they work on the main repo, and any fork.

e.g.
```markdown
* [Installation](#installation)
```